src/components/CardPokemon/index.vue

```diff
@@ -1,26 +1,43 @@
 <template>
-  <article class="relative pt-5 pb-3 rounded-lg shadow flex flex-col items-center transition ease ring-2 ring-transparent hover:ring-emerald-500 hover:ring-offset-2 bg-green-700 bg-opacity-20">
-    <div class="absolute leading-none top-[2px] left-[2px] rounded-tl-lg rounded py-1 px-2 flex items-center bg-green-500 bg-opacity-60">
-      <span class="text-xs text-black">#{{ pokemon.id }}</span>
+  <article :class="('relative pt-5 pb-3 rounded-lg shadow flex flex-col items-center transition ease ring-2 ring-transparent hover:ring-emerald-500 hover:ring-offset-2 bg-opacity-20 bg-'+types[0].type.name)">
+    <div class="absolute leading-none top-[2px] left-[2px] rounded-tl-lg rounded py-1 px-2 flex items-center bg-gray-100 bg-opacity-60">
+      <span class="text-xs text-black">#{{ id }}</span>
     </div>
     <img
-      :src="imageUrl + pokemon.id + '.png'"
-      :alt="pokemon.name + ' image'"
+      :src="imageUrl + id + '.png'"
+      :alt="name + ' image'"
```

Comment:
improvement suggestion

we usually put this kind of param/input into some `methods` or `computed` to make it more maintainable

so it'll be like this
```vue
<img
  :src="imageSource"
  :alt="alt"
  height="140"
  width="140"
  class="h-auto w-36"
>
```
```js
computed: {
	imageSource() {
		return `${this.imageUrl}${this.id}.png`
	},
	alt() {
		return `${this.name} image`
	}
}
```

Reply:
Oh I see, thank you @naufalihsank for the suggestion. I've fixed it [here](https://github.com/sheilaazhar/fe-onboarding-mini-task/pull/5/commits/41400af63db39d5a506de460972d07094d11287c)